src/Kanvas/AccessControlList/Actions/CreateRoleAction.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+namespace Kanvas\AccessControlList\Actions;
+
+use Bouncer;
+use Silber\Bouncer\Database\Role as SilberRole;
+
+class CreateRoleAction
+{
+    /**
+     * __construct
+     *
+     * @return void
+     */
+    public function __construct(
+        public string $name,
+        public string $title,
+    ) {
+    }
+
+    /**
+     * execute
+     *
+     * @return SilberRole
+     */
+    public function execute(): SilberRole
+    {
+        $role = Bouncer::role()->firstOrCreate([
+            'name' => $this->name,
+            'title' => $this->title,
```

Comment:
@FredPeal do we need to specify the app or company?

Reply:
No